web/src/stream_pill.ts

```diff
@@ -12,29 +12,41 @@ export type StreamPill = {
     stream_sub_count?: number;
 };
 
-type StreamPillWidget = InputPillContainer<StreamPill>;
+export type StreamPillWidget = InputPillContainer<StreamPill>;
 
 export type StreamPillData = StreamSubscription & {type: "stream"};
 
-function stream_display_pill(sub: StreamSubscription): string {
+export function stream_display_pill(sub: StreamSubscription): string {
```

Comment:
I think this change is not needed anymore.

Reply:
Yeah right.